website/_plugins/div_tag.rb

```
@@ -0,0 +1,17 @@
+module Jekyll
+  class DivTag < Liquid::Block
+    def initialize(tag_name, markup, tokens)
+      super
+      @class = markup.strip
+    end
+
+    def render(context)
++      site = context.registers[:site]
```

RuboCop (1.84.2)

[warning] Operator `+@` used in void context.

(Lint/Void)